hyp3_gamma/insar/etc/README_InSAR_GAMMA.txt

```diff
@@ -128,6 +129,14 @@ Two look vector maps are output with the product as GeoTIFFs. The lv_theta map i
 
 *Note that these files are optional. Select the "Angle Maps" option in the HyP3 Processing Options to include them in the product package.*
 
+----------------
+## 9. Incidence Angle Map
+
+The local incidence angle is defined as the angle between the incident radar signal and the local surface normal.  Output is a geotiff in radians giving the local incidence angle for each pixel in the scene.
+
+*Note that this file is optional. Select the "Incidence angle" option in the HyP3 Processing Options to include it in the product package.*
```

Comment:
If we use the same labels as we did for RTC, the option the user will see in Vertex is `Include Inc. Angle Map`

![Screenshot from 2021-04-23 08-33-07](https://user-images.githubusercontent.com/17994518/115902172-9caccd80-a40e-11eb-8ea8-91c191564fe3.png)

Reply:
And the option in the API/SDK would be `include_inc_map` .  I'm open to revising those if needed, but I think they've served us reasonably well for RTC.